Tests/FluentMySQLTests/FluentMySQLTests.swift

```diff
@@ -206,6 +207,56 @@ class FluentMySQLTests: XCTestCase {
         test = try test.save(on: conn).wait()
     }
 
+    func testReferences() throws {
+        let conn = try benchmarker.pool.requestConnection().wait()
+        defer { benchmarker.pool.releaseConnection(conn) }
+
+        // Prep tables
+        defer { try? Child.revert(on: conn).wait()
+                try? Parent.revert(on: conn).wait() }
+        try Parent.prepare(on: conn).wait()
+        try Child.prepare(on: conn).wait()
+        MySQLDatabase.enableLogging(conn.logger!, on: conn)
+        // Save Parent
+        var parent = Parent(id: 64, name: "Snuffles")
+        parent = try parent.save(on: conn).wait()
```

Comment:
Tried using save and create on the Parent....save doesn't work because it has an ID, create ignores my ID and creates a new Parent with ID 0 🤷‍♂️